src/test/java/BelhardSite.java

```diff
@@ -0,0 +1,43 @@
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+public class BelhardSite {
+    WebDriver webdriver;
+
+    @BeforeTest
+    public void setup() {
+        System.setProperty("webdriver.chrome.driver", "C:\\WebDrivers\\chromedriver.exe");
```

Comment:
Please get rid of this

Reply:
Can't not be eliminated. Strange behaviour. Will be resolved later